### PR TITLE
Feature/refactor filenames and imports

### PR DESCRIPTION
**Pull request by clewis1**
https://companieshouse.atlassian.net/browse/IDVA3-1278

### CODE REVIEW

File: test/routers/healthcheck.unit.ts
Comment (PhilipGlover) [inline test/routers/healthcheck.unit.ts:1]: I don't think we need this test at all - in ECS the start page is used as a healthcheck.  And the 200 status is already tested in `start.unit`

File: src/app.ts [hunk 1]
```diff
@@ -1,11 +1,11 @@
...
+import cookieParser from "cookie-parser";
+import express, { NextFunction, Request, Response } from "express";
 import nunjucks from "nunjucks";
 import path from "path";
-import routerDispatch from "./router.dispatch";
-import { servicePathPrefix, ExternalUrls } from "./constants";
-import { sessionMiddleware } from "./middleware/session";
-import cookieParser from "cookie-parser";
+import { ExternalUrls, servicePathPrefix } from "./constants";
 import { logger } from "./lib/Logger";
```
Comment (PhilipGlover) [inline src/app.ts:6]: this path needs to be `./lib/logger` now - lower case 'l'

File: src/routers/confirmCompanyRouter.ts [hunk 2]
```diff
@@ -1,12 +1,12 @@
...
+import { Transaction } from "@companieshouse/api-sdk-node/dist/services/transaction/types";
 import { NextFunction, Request, Response, Router } from "express";
-import { ConfirmCompanyHandler } from "./handlers/confirmCompany/confirmCompany";
-import { handleExceptions } from "../utils/async.handler";
 import { PrefixedUrls } from "../constants";
-import { postTransaction } from "../services/internal/transaction.service";
-import { Transaction } from "@companieshouse/api-sdk-node/dist/services/transaction/types";
-import { PscVerification } from "@companieshouse/api-sdk-node/dist/services/psc-verification-link/types";
-import { createPscVerification } from "../services/internal/pscVerificationService";
 import { logger } from "../lib/Logger";
```
Comment (PhilipGlover) [inline src/routers/confirmCompanyRouter.ts:5]: `/logger` here too and in several other files

File: src/routers/handlers/personal-code/personalCode.ts
Comment (PhilipGlover) [inline src/routers/handlers/personal-code/personalCode.ts:1]: the import in the personalCode router that uses this handler will need updating.

**Review by PhilipGlover — CHANGES_REQUESTED** [review]
not all imports have been amended, so service won't build and start

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

File: src/types/jsonData.ts [hunk 3]
```diff
@@ -1,12 +1,14 @@
-type JSONValue =
-    | string
-    | number
-    | boolean
-    | JSONObject
-    | JSONArray;
+/* eslint-disable no-use-before-define */
```
Comment (PhilipGlover) [inline src/types/jsonData.ts:1]: this does the beg question whether this should be kept if it's unused.  This has come from the web-starter project. 
Are we going to be using raw JSON in the web?

Comment (PhilipGlover) [reply src/types/jsonData.ts]: file removed

**Review by PhilipGlover — APPROVED** [review]
as per comments above

**Pull request merged**